src/main/webapp/app/exam/participate/exam-cover/exam-participation-cover.component.ts

```diff
@@ -149,9 +150,11 @@ export class ExamParticipationCoverComponent implements OnChanges, OnDestroy, On
             this.examParticipationService.saveStudentExamToLocalStorage(this.exam.course!.id!, this.exam.id!, this.studentExam);
             this.onExamStarted.emit(this.studentExam);
         } else {
+            this.waitingForExamFetching = true;
             this.examParticipationService
```

Comment:
I recognised we don't unsubscribe from the subscription. Can you add this, too?  😇